content/docs/authorization/release/_index.md

```diff
@@ -6,19 +6,21 @@ Description: >
   Dell Container Storage Modules (CSM) release notes for authorization
 ---
 
-## Release Notes - CSM Authorization 1.3.0
+## Release Notes - CSM Authorization 1.4.0
 
 ### New Features/Changes
 
-- [CSM-Authorization can deployed with helm](https://github.com/dell/csm/issues/261)
+- CSM 1.4 Release specific changes. ([#350](https://github.com/dell/csm/issues/350))
+- Implementation: enable authorization for csm observability powerscale. ([#413](https://github.com/dell/csm/issues/413))
```

Comment:
This is an observability related release note. No changes were made to Authorization in this feature.

Reply:
oh okay ! Will create another PR to make this change .